src/index.js

```diff
@@ -0,0 +1,13 @@
+import './style.css';
+import UserInterface from './list.js';
+
+// class ToDoList{
+//     constructor(description, completed ,index){
+//         this.description = description;
+//         this.completed = completed;
+//         this.index= index;
+//     }
+// }
```

Comment:
- [ ] Kindly remove the commented code, its not good practise to have commented code in your codebase.